pkg/compactor/bucket_compactor.go

```diff
@@ -670,6 +679,14 @@ func NewBucketCompactorMetrics(blocksMarkedForDeletion prometheus.Counter, reg p
 			Name: "cortex_compactor_group_compactions_total",
 			Help: "Total number of group compaction attempts that resulted in new block(s).",
 		}),
+		blockCompactionDelay: promauto.With(reg).NewHistogramVec(prometheus.HistogramOpts{
+			Name:                            "cortex_compactor_block_compaction_delay_minutes",
```

Comment:
It can be unnecessary in this case, but [Prometheus metrics use base units by convention](https://prometheus.io/docs/practices/naming/#base-units).

Reply:
Converted these to seconds